README.md

```diff
@@ -56,6 +56,7 @@ A list of every a11y concern Checka11y.css will check for and highlight:
   - `<select>`
   - `<textarea>`
   - `<video controls>`
+- Check the dir attribute has the correct values
```

Comment:
- Checks the `dir` attribute is only set to `ltr`, `rtl` or `auto`

Could you add the backticks for the code?